tests/test_states_group_navigation.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+
+from aiogram.contrib.fsm_storage.memory import MemoryStorage
+from aiogram.dispatcher import FSMContext
+from aiogram.dispatcher.filters.state import StatesGroup, State
+
+
+class FakeDispatcher:
+    context = FSMContext(MemoryStorage(), chat=1, user=1)
+    
+    @classmethod
+    def get_current(cls, no_error=True):
+        return cls
+
+    @classmethod
+    def current_state(cls):
+        return cls.context
+
+    @classmethod
+    def reset(cls):
+        cls.context = FSMContext(MemoryStorage(), chat=1, user=1)
+
+
+@pytest.fixture
+async def reset_dispatcher():
+    FakeDispatcher.reset()
+    context = FakeDispatcher.get_current().current_state()
+    current_state = await context.get_state()
+    assert  current_state == None
+    yield FakeDispatcher
+
+
+class MyGroup(StatesGroup, dispatcher=FakeDispatcher()):
+    state_1 = State()
+    state_2 = State()
+
+
+class TestNavigation:
+    @pytest.mark.asyncio
```

Comment:
Better use `pytestmark = pytest.mark.asyncio` in global module namespace or mark entire class with this decorator

https://docs.pytest.org/en/stable/example/markers.html#marking-whole-classes-or-modules